pkg/registry/generic/rest/streamer.go

```diff
@@ -56,6 +59,16 @@ func (s *LocationStreamer) InputStream(apiVersion, acceptHeader string) (stream
 	if err != nil {
 		return
 	}
+
+	if resp.StatusCode == http.StatusInternalServerError {
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
This is incredibly dangerous and is not safe for us to do.  This could blow up the master if a client tricks the master into accessing a stream of unlimited bytes